biz.aQute.bndlib/src/aQute/bnd/osgi/Verifier.java

```diff
@@ -531,6 +532,22 @@ public void verify() throws Exception {
 		verifyCapabilities();
 		verifyMetaPersistence();
 		verifyPathNames();
+
+		doVerifierPlugins();
+	}
+
+	private void doVerifierPlugins() {
+		for (VerifierPlugin plugin : getPlugins(VerifierPlugin.class)) {
+			try {
+				Processor previous = beginHandleErrors(plugin.toString());
+				plugin.verify(analyzer);
+				endHandleErrors(previous);
```

Comment:
Shall I fix this one only for the sake of this PR?
Or shall I fix both?
Or note it and fix the issue as a follow-on?